railties/test/application/rake/dbs_test.rb

```diff
@@ -68,6 +68,60 @@ def db_create_and_drop(expected_database, environment_loaded: true)
         db_create_and_drop("db/development.sqlite3", environment_loaded: false)
       end
 
+      test "db:create and db:drop don't raise errors when loading YAML with multiline ERB" do
+        app_file "config/database.yml", <<-YAML
+          development:
+            database: <%=
+              Rails.application.config.database
+            %>
+            adapter: sqlite3
+        YAML
+
+        app_file "config/environments/development.rb", <<-RUBY
+          Rails.application.configure do
+            config.database = "db/development.sqlite3"
+          end
+        RUBY
+
+        db_create_and_drop("db/development.sqlite3", environment_loaded: false)
+      end
+
+      test "db:create and db:drop don't raise errors when loading YAML containing conditional statements in erb" do
```

Comment:
I think the usage of `ERB` word in the test description should be consistent for all the test cases. 

First test(https://github.com/rails/rails/pull/35583/files#diff-acebf6e9e1ac03337e62ff6ececec15dR71) has `ERB` all in capital 
Whereas here and the last test has `erb`

Reply:
Thank you for catching that. It's been fixed.